pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -539,6 +461,202 @@ var _ = Describe("Pod Network", func() {
 			})
 		})
 	})
+
+	Context("Masquerade startDHCP", func() {
+		It("should succeed", func() {
```

Comment:
I don't get the context of these two `It`s. Could you  put them in more specific context? When it should fail and when it should succeed? The same for all the other tests.